packages/remirror__extension-caption/src/figcaption-extension.ts

```diff
@@ -0,0 +1,41 @@
+import {
+  ApplySchemaAttributes,
+  extension,
+  ExtensionTag,
+  NodeExtension,
+  NodeExtensionSpec,
+  NodeSpecOverride,
+} from '@remirror/core';
+
+export interface FigcaptionOptions {}
+
+/**
+ * An extension to annotate content with a caption
+ */
+@extension<FigcaptionOptions>({})
+export class FigcaptionExtension extends NodeExtension<FigcaptionOptions> {
+  get name() {
+    return 'figcaption' as const;
+  }
+
+  readonly tags = [ExtensionTag.Block];
+
+  createNodeSpec(extra: ApplySchemaAttributes, override: NodeSpecOverride): NodeExtensionSpec {
+    return {
+      content: 'paragraph',
```

Comment:
Ideally this would just be a text node, but I can't get that to work. This seems to be the only way?

If it were an inline node, it would work well with the existing Image extension default, however it needs to be a block node to contain a paragraph (and hence text)

This also means that Image needs to become a block node, to avoid the "mixing block and inline nodes" error